src/merge.js

```diff
@@ -19,6 +19,9 @@ var _extend = require('./internal/_extend');
  *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': 40 });
  *      //=> { 'name': 'fred', 'age': 40 }
  *
+ *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': undefined });
+ *      //=> { 'name': 'fred', 'age': undefined }
```

Comment:
I'm not sure what you mean. Are you suggesting that the added docs should be moved (and reworked if applicable) to `R.dissoc`?  The purpose of this commit is to clarify the behavior of `R.merge` with respect to any items of higher precedence that contain fields set to `undefined`.  Should this behavior be left undocumented?  Or do you mean something else?